app/queries/tokens_query.rb

```diff
@@ -1,21 +1,55 @@
 class TokensQuery
   def initialize(relation = JwtApiEntreprise.all)
     @relation = relation
+
+    self.relevent
```

Comment:
Étrange d'effectuer un call SQL caché ici dans l'initializer. 

Après avoir réfléchi j'ai pas mieux à proposer en vrai.. mais je suis vraiment pas fan. @brindu peut-être ?

Reply:
Au passage, c'est pas `relevant` et non `relevent` ?

Reply:
ce n'est pas un call sql. c'est le loading dans la relation ActiveRecord, aucun call n'est effectué. Le vrai call en base n'est effectué que si tu appelles des méthodes type .each, .count, .all. 

Oui pour l'orthographe

Reply:
<img width="1420" alt="Screen Shot 2021-09-22 at 10 41 36" src="https://user-images.githubusercontent.com/1279968/134311583-cb923c62-272a-4c9c-8570-beedc8adad05.png">

Chez moi un `.new` tout seul effectue le call --^

Reply:
C'est une particularite de la console. Il effectue un inspect de ce que tu tapes en invite ruby et ca trigger la requete sql. Sneaky and nasty

Reply:
Tu peux faire un `TokensQuery.new; nil` et il fait plus la requete

Reply:
Ah oui en effet, j'ai pas pensé au inspect.

Je solve, tant pis.